domino-ui/src/main/java/org/dominokit/domino/ui/datatable/plugins/GroupingPlugin.java

```diff
@@ -96,6 +99,26 @@ public void appendRow(DataTable<T> dataTable, TableRow<T> tableRow) {
         }
     }
 
+    public void expandAllGroups() {
+        for (DataGroup<T> dataGroup : dataGroups.values()) {
+            if (!dataGroup.expanded)
+            {
```

Comment:
Please stick to the project formatting here.
Maybe we should provide a format style in the project itself, for now please keep the open brace in the same if statement line.